OpenEdition Journals.js

```diff
@@ -28,96 +48,92 @@ function detectWeb(doc, url) {
 		}
 	}
 
-	if (ZU.xpath(doc, '//div[@id="inside"]/div[@class="sommaire"]\
-			/dl[@class="documents"]/dd[@class="titre"]/a').length ||
-		ZU.xpath(doc, '//ul[@class="summary"]//div[@class="title"]/a').length) {
+	if (ZU.xpath(doc, '//div[@id="inside"]/div[@class="sommaire"]/dl[@class="documents"]/dd[@class="titre"]/a').length
+		|| ZU.xpath(doc, '//ul[@class="summary"]//div[@class="title"]/a').length) {
 		return "multiple";
-	} else if (ZU.xpath(doc, '//h1[@id="docTitle"]/span[@class="text"]').length ||
-		url.match(/document\d+/)) {
+	}
+	else if (ZU.xpath(doc, '//h1[@id="docTitle"]/span[@class="text"]').length
+		|| url.match(/document\d+/)) {
 		return "journalArticle";
 	}
+	return false;
 }
 
 function scrape(doc, url) {
-	//is this still necessary??
-	if (url.match(/persee\-\d+/)) {
+	var translator;
```

Comment:
This is better handled by using `let translator = ...`  in each individual condition for clarity

Reply:
It's done